pkgs/development/compilers/unison/default.nix

```
@@ -1,23 +1,23 @@
 { lib, stdenv, fetchurl, autoPatchelfHook
-, ncurses5, zlib, gmp
+, ncurses6, zlib, gmp
```

```suggestion
, ncurses, zlib, gmp
```

I prefer setting ncurses version in `pks/top-level/all-packages.nix` like this:

```
  unison-ucm = callPackage ../development/compilers/unison {
    ncurses = ncurses6;
  };
```

What do you think?

---

Are there any recommendations? In this case, if you choose the wrong ncurses, the auto patch elf hook will fail. 

In that case, IMO the visibility of the problem gets lost on the derivation and you’d have to go to all-packages to understand what the issue is.

I am not against changing if, but I just hit this issue bumping unison in this PR, and it was straightforward to identify the issue.